_posts/2022-07-19_UK_workshops

```diff
@@ -0,0 +1,26 @@
+---
+ layout: post
+ title: MDAnalysis + Machine Learning Workshops 
+ ---
+ 
+ In collaboration with CCPBioSim, 3 MDAnalysis core devs (@ialibay, @micaela-matta, @richardjgowers) in collaboration with [Dr Antonia Mey](https://mey-research.org) (University of Edinburgh) and [Prof. Matteo Degiacomi](https://degiacomi.org) (University of Durham) organised and ran a 2-day hybrid (in person/online) workshop titled “MDAnalysis and Machine Learning for Molecular Simulations” on June 9-10 2022, immediately following the [8th Annual CCPBioSim Conference](https://web.cvent.com/event/7f574af0-1953-4a86-ab63-6334c7705ac8/summary).
+
+The topics covered included: 
+- The fundamentals and basics of MDAnalysis
+- How to get started with machine learning and tools such as scikit-learn
+- How to use MDAnalysis in conjunction with machine learning
+- Applying these tools to your own research projects
+
+The workshop was hosted at the University of Edinburgh and consisted of 4 half days in a hybrid format, with over 40 in person attendees and over 30 attendees online at any point in time. A special thank you to Jasmin Güven (PhD student in the group of Antonia Mey) who moderated questions from online participants!
+ 
+The teaching materials can be found on [GitHub](https://github.com/MDAnalysis/WorkshopMDMLEdinburgh2022).
```

Comment:
maybe mention they're runnable via colab to make it more appealing to get clicks?